### PR TITLE
Readme for testing phase

### PR DESCRIPTION
**Pull request by simesy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,16 +2,81 @@
...
-* Starts a local mail Mail Transfer Agent (MTA) in order to test and view mails
+We welcome testers of this tool. You will probably be an existing user of Pygmy who
+can verify the same functionality, or perhaps who has had trouble installing Pygmy in the
+past on Windows.
 
-You can find more documentation here: [https://pygmy.readthedocs.io](https://pygmy.readthedocs.io)
+## Handling the old Pygmy
+
+You should run `pygmy down` to ensure that Pygmy is not running. It is not possible
+to run both together using the default configuration.
```
Comment (fubarhouse) [inline README.md]: I genuinely feel this paragraph should be removed - I will run a test and see.
Original premise used a lot of compatibility - I'll check and report back

**Review by fubarhouse — COMMENTED** [review]
LGTM - I will double-check compatibility though.
Container config is identical and the resolvr differences are minor if not compatible.

File: README.md (hunk 1 above)
Comment (simesy) [reply README.md]: Please modify to taste. Most of the value is in me giving you something to chew over :D

Comment (fubarhouse) [reply README.md]: Confirmed. Please remove this paragraph and I'll approve.

Comment (simesy) [reply README.md]: Thinking about it, my worry is that if people are still running the old pygmy they might think the new pygmy is working 🤔 but it's just the old one.

**Review by fubarhouse — APPROVED** [review]

**Pull request merged**